lib/devicetrust/authn/authn_test.go

```diff
@@ -61,38 +62,34 @@ func TestCeremony_Run(t *testing.T) {
 	}
 
 	tests := []struct {
-		name  string
-		dev   testenv.FakeDevice
-		certs *devicepb.UserCertificates
+		name      string
+		dev       testenv.FakeDevice
+		certs     *devicepb.UserCertificates
+		sshCert   []byte
+		sshSigner *keys.PrivateKey
```

Comment:
sshSigner unused?

Reply:
Yeah it was, in the latest commit upgraded this test to actually do the ssh challenge signature and verify it